.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: rust
 sudo: required
 dist: trusty
+group: deprecated-2017Q4
```

Comment:
Can you put a comment here about why this is needed? (is it even needed after the convertor fix?)

Reply:
Addressed. It's not necessary.